Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -20,12 +20,22 @@
 
 #include "Firestore/core/src/firebase/firestore/model/database_id.h"
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
+#include "Firestore/core/src/firebase/firestore/util/log.h"
 
 using firebase::firestore::model::DatabaseId;
 using firebase::firestore::remote::Serializer;
 
 namespace {
 
+// A list of targets to fuzz test. Should be kept in sync with the method
+// GetFuzzingTarget().
+enum FuzzingTarget { NONE = 0, SERIALIZER = 1 };
```

Comment:
A couple of nits:
* in general, prefer `enum class` to `enum`, which 
* style guide has a slight preference for constant-style names (`kNone`) instead of macro-style (`NONE`). I also think that macro-style ALL_CAPS are best avoided when possible;
* the values of enumeration members are well-defined. Unless you want to change the defaults, I think it's best to omit explicit values, due to redundancy.